File: 6.php
<?php

function find() {
    $sumOfSquares = 0;
    $sum = 0;
    for ($i = 1; $i <= 100; $i++) {
        $sumOfSquares += pow($i, 2);
        $sum += $i;
    }

    return pow($sum, 2) - $sumOfSquares;
}

var_dump(find()); die;

File: 6.go
package main

import "fmt"
import "math"

func main() {
    fmt.Println(sumSquareDiff());
}

func sumSquareDiff() (int) {
    sumOfSquares := 0
    sum := 0

    for i := 1; i <= 100; i++ {
        sumOfSquares += int(math.Pow(float64(i), 2))
        sum += i
    }

    return int(math.Pow(float64(sum), 2)) - sumOfSquares
}
